File: LeetCode_1925/Main.cs
public class Solution {
    public int CountTriples(int n) {
        int res = 0;

        for (int a = 1; a <= n; a++) {
            for (int b = 1; b <= n; b++) {
                int c = (int)Math.Sqrt(a * a + b * b);
                if (c <= n && c * c == a * a + b * b) {
                    res++;
                }
            }
        }

        return res;
    }
}

File: LeetCode_1925/Main.py
import math
class Solution(object):
    def countTriples(self, n):
        """
        :type n: int
        :rtype: int
        """
        res = 0

        for a in range(1, n + 1):
            for b in range(1, n + 1):
                c = int(math.sqrt(a * a + b * b))
                if c <= n and c * c == a * a + b * b:
                    res += 1

        return res